spec/services/participants/withdraw/npq_spec.rb

```diff
@@ -12,7 +12,7 @@
       cpd_lead_provider: cpd_lead_provider,
       participant_id: npq_profile.user.id,
       course_identifier: "npq-leading-teaching",
-      reason: "left-teaching-profession",
+      reason: Participants::Withdraw::NPQ.reasons,
```

Comment:
Does this need to be a sample?

Reply:
that's very interesting that this test then 😲 ! Will look into that